packages/sdk/src/types/config.ts

```diff
@@ -0,0 +1,71 @@
+export enum Environment {
+  DEVNET = 'devnet',
+  TESTNET = 'testnet',
+  MAINNET = 'mainnet',
+}
+
+export type Domain = {
+  id: number;
+  name: string;
+};
+
+export enum ResourceType {
+  ERC20 = 'erc20',
+  ERC721 = 'erc721',
+  PERMISSIONED_GENERIC = 'permissionedGeneric',
+  PERMISSIONLESS_GENERIC = 'permissionlessGeneric',
+}
```

Comment:
Any reason for the use of enums over string literals here?

Reply:
I think this is more expressive in our case where this will be passed to functions so you already have predefined values that can be passed as params.